### DISCUSSION TITLE
GitHub integration is disable in Codacy

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#2816] [Ideas]
[GitHub integration is disabled and the information of code is old.](https://app.codacy.com/gh/pyvista/pyvista/dashboard) Is it possible to update?
![image](https://user-images.githubusercontent.com/7513610/174412180-ea6883e2-7078-4672-a0df-2bf2d69837ae.png)

**Comment by banesullivan** [comment]
I updated the webhook - this should update shortly

**Reply by tkoyama010** [reply]
Great thanks!

**Reply by tkoyama010** [reply]
I am glad that we got an "A" in codacy.
![image](https://user-images.githubusercontent.com/7513610/176341825-943945dc-8d34-4a86-989d-f490e3ed8b5e.png)
